skills/instagram-warmup/README.md

```
@@ -0,0 +1,47 @@
+
+Expert Instagram account warmup guide — 14-day engagement plans, Stories trust-building strategy, first Reel warmup content briefs, carousel formats, and post-warmup assessment for new and purchased accounts before posting promotional content.
+
+## Installation
+
+```bash
+npx skills add genfeedai/skills/instagram-warmup
+```
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)